config/log.go

```diff
@@ -0,0 +1,14 @@
+package config
+
+type LogConfig struct {
+	logPath  string
+	logLevel string
+}
+
+func (dc LogConfig) LogPath() string {
+	return dc.logPath
```

Comment:
I guess you missed on reading this config value

Reply:
We haven't used logger anywhere in the app. Should we include it? Or should we remove this code?

Reply:
Okay. Will take care of this now on.